client/src/app/bookings/layout.tsx

```diff
@@ -11,17 +11,30 @@ import { Footer } from "@/components/generic/Footer/Footer"
 import { QueryClientProvider } from "@tanstack/react-query"
 import queryClient from "@/services/QueryClient"
 
-type IBookingLayout = Readonly<{ children: ReactNode }>
+type IBookingLayout = Readonly<{
```

Comment:
Some of the tests fail due to "Type error: Layout "src/app/bookings/layout.tsx" has an invalid "default" export:
  Type "Readonly<{ children: ReactNode; policyInfoProps: Omit<ReactNode, "handbookBookLodgeClick"> & { children?: ReactNode; }; }>" is not valid." does this mean I should remove the policyInfoProps and just leave it as children or should I add policyInfoProps to the Innerbookinglayout function's argument?

Reply:
your issue here is `Omit<React.ReactNode, "handbookBookLodgeClick">` read the documentation for omit https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys, to undersstand that your type for `policyInfoProps` is invalid.

my guess is this code is copy pasted